src/ServiceManagement/Compute/Commands.ServiceManagement/IaaS/Extensions/Chef/SetAzureVMChefExtension.cs

```diff
@@ -107,9 +107,10 @@ public class SetAzureVMChefExtensionCommand : VirtualMachineChefExtensionCmdletB
         [Parameter(
             ValueFromPipelineByPropertyName = true,
             HelpMessage = "Configures the chef-client service for unattended execution. The node platform should be Windows." +
-                          "Options: 'none' or 'service'." +
+                          "Allowed options: 'none', 'service' and 'task'" +
                           "none - Currently prevents the chef-client service from being configured as a service." +
-                          "service - Configures the chef-client to run automatically in the background as a service.")]
+                          "service - Configures the chef-client to run automatically in the background as a service." +
+                           "task - Configures the chef-client to run automatically in the background as a secheduled task.")]
         [ValidateNotNullOrEmpty]
```

Comment:
@NimishaS can we add a ValidateSet attribute for this. That will drastically reduce a lot of code that is written to verify if it has the right values, plus would avoid non-supported values.
https://msdn.microsoft.com/en-us/library/system.management.automation.validatesetattribute(v=vs.85).aspx